client.go

```diff
@@ -745,3 +774,8 @@ func (c *Client) logStderr(r io.Reader) {
 	// Flag that we've completed logging for others
 	close(c.doneLogging)
 }
+
+func isJSON(str string) bool {
```

Comment:
It might be better to actually return `*json.RawMessage` (nil meaning not valid JSON) and then just unmarshaling that into the final destination structure to avoid the double-JSON parse.